packages/contracts/migrations/development/Deploy001_Diamond_Dollar.s.sol

```diff
@@ -0,0 +1,308 @@
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfAccessControlFacet
+            })
+        );
+        cuts[1] = (
+            FacetCut({
+                facetAddress: address(diamondCutFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfDiamondCutFacet
+            })
+        );
+        cuts[2] = (
+            FacetCut({
+                facetAddress: address(diamondLoupeFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfDiamondLoupeFacet
+            })
+        );
+        cuts[3] = (
+            FacetCut({
+                facetAddress: address(managerFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfManagerFacet
+            })
+        );
+        cuts[4] = (
+            FacetCut({
+                facetAddress: address(ownershipFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfOwnershipFacet
+            })
+        );
+        cuts[5] = (
+            FacetCut({
+                facetAddress: address(twapOracleDollar3PoolFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfTWAPOracleDollar3poolFacet
+            })
+        );
+        cuts[6] = (
+            FacetCut({
+                facetAddress: address(ubiquityPoolFacetImplementation),
+                action: FacetCutAction.Add,
+                functionSelectors: selectorsOfUbiquityPoolFacet
+            })
+        );
+
+        // deploy diamond
+        diamond = new Diamond(diamondArgs, cuts);
+
+        // stop sending owner transactions
+        vm.stopBroadcast();
+
+        //=======================
+        // Diamond permissions
+        //=======================
+
+        // start sending admin transactions
+        vm.startBroadcast(adminPrivateKey);
+
+        AccessControlFacet accessControlFacet = AccessControlFacet(
+            address(diamond)
+        );
+
+        // grant diamond dollar minting and burning rights
+        accessControlFacet.grantRole(
+            DOLLAR_TOKEN_MINTER_ROLE,
+            address(diamond)
+        );
+        accessControlFacet.grantRole(
+            DOLLAR_TOKEN_BURNER_ROLE,
+            address(diamond)
+        );
+
+        // stop sending admin transactions
+        vm.stopBroadcast();
+
+        //=========================
+        // UbiquiPoolFacet setup
+        //=========================
+
+        // start sending admin transactions
+        vm.startBroadcast(adminPrivateKey);
+
+        UbiquityPoolFacet ubiquityPoolFacet = UbiquityPoolFacet(
+            address(diamond)
+        );
+
+        // add collateral LUSD token
+        uint256 poolCeiling = 10_000e18; // max 10_000 of collateral tokens is allowed
```

Comment:
I see you reduced the collateral pool ceiling to 10k tokens, I am fine with that. Perhaps we should add this information to readme, also giving a short explanation.

Reply:
but i think this is more like reserved value, the amount is a internal consensus as of now, unless bound to change for future reference on users side

Reply:
The pool ceiling param is [adjustable](https://github.com/ubiquity/ubiquity-dollar/blob/904396c1596128bb8f4b3bffde0ccb1f555a8451/packages/contracts/src/dollar/facets/UbiquityPoolFacet.sol#L162) so we can update it even after the deployment. We don't have a consensus regarding the value, we'll set the final value only on the "deployment day".